pkg/v1/providers/config_default.yaml

```diff
@@ -174,9 +174,9 @@ NSXT_PASSWORD: ""
 #! NSX-T host
 NSXT_MANAGER_HOST: ""
 #! set this to true if NSX-T uses self-signed cert
-NSXT_ALLOW_UNVERIFIED_SSL: "false"
+NSXT_ALLOW_UNVERIFIED_SSL: false
```

Comment:
Why the type was changed? For customer that upgrade to newer version, their config would still be string type, wouldn't it breaks their upgrade?